### ISSUE TITLE
Name of the iScroll object created by fullpage.js

### ISSUE THREAD

**Issue by Stefanowitsch84** [alvarotrigo/fullPage.js#2279]
I want to remove the iscroll function when visiting the website on a mobile application.

In the fullpage initialisation I want to do it like that:

`afterResponsive: function(isResponsive){
					scroller.destroy();
					scroller = null;
				}`

But I need to know the object name of the iScroll instance that is created by the fullpage script.

**Comment by alvarotrigo** [comment]
Turning scrollOverflow to `false` is not yet possible as stated here: https://github.com/alvarotrigo/fullPage.js/issues/958

But if you want to do it by yourself, just take a look [at the code](https://github.com/alvarotrigo/fullPage.js/blob/master/jquery.fullPage.js#L2968) for it.

You can also take a look at [my answer here](http://stackoverflow.com/a/39369282/1081396) and will give you some hints about it.

Basically:
```javascript
 var iscroll = $('.fp-section.active').find('.fp-scrollable').data('iscrollInstance');
```
